README.md

```diff
@@ -27,3 +27,20 @@ In case you want to use it without SUSE Manager:
 3. At the moment `db_backend` can have only two values: `oracle` or `postgresql`.
    Therefore it will load `oraclegate` or `postgresqlgate` module. Once you want
    something else, like H2, you suppose to create `h2gate` module.
+
+## How to create a release
+
+* Update version in setup.py and src/smdba/smdba (as an example see
+https://github.com/SUSE/smdba/commit/530731003cbf3a275f7fe3bfb97b4337f975a38e
+
+* After commiting it to the master branch, create a tag and push it:
+ ```git tag X.Y.Z && git push --tags``` where X.Y.Z is the new version
+
+* Call build-src.sh to create the tarball
+
+* Update the tarball in:
+
+https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Stable/smdba
```

Comment:
Wrong project. Submissions go to Master. Things into Stable from Master when we promote for a release.

What about changing the SPEC version and updating the changelog?

Also we should mention that backporting to Devel:Galaxy:Manager:Head is required, and most likely to released versions of SUSE Manager.